src/main.rs

```diff
@@ -21,6 +25,17 @@ use reqwest::redirect::Policy;
 use url::Url;
 use utils::body_from_stdin;
 
+fn get_user_agent() -> &'static str {
```

Comment:
I usually prefer to move any function that doesn't belong in any module to `utils.rs`. What do you think?

Reply:
I think this one is best off in the same module as where it's used, since it isn't used anywhere else. That also means it can stay private.

But I would also be ok with moving it, if you'd like.